app-emulation/rkt/files/rkt-gc.service

```diff
@@ -1,7 +0,0 @@
-[Unit]
```

Comment:
Your PR preserves the 0.7 ebuild which needs these. Either drop the old ebuild entirely or keep these files in place for now.